src/cmdstan/arguments/arg_output_sig_figs.hpp

```diff
@@ -12,15 +12,15 @@ class arg_output_sig_figs : public int_argument {
     _description
         = "The number of significant figures used for the output CSV files.";
     _validity
-        = "0 <= integer <= 18 or -1 to use the default number of significant "
-          "figures";
-    _default = "-1";
-    _default_value = -1;
+        = "0 <= integer <= 18, or -1 to use the system library's default "
```

Comment:
How about "an integer number of significant figures between 0 and 18 inclusive, or -1 to use the default (8)"